pom.xml

```diff
@@ -701,21 +707,24 @@
                 <volumes>
                   <bind>
                     <volume>${image.root}:/images/</volume>
+                    <volume>bucketeer-test-write-csv:/csv/</volume>
```

Comment:
I'm not clear on why you're mounting the volume since you're using the `docker cp`. From the application's perspective, it's just writing to a file system. From the test's perspective, it's copying a file from the container to the host and then checking it. The mounted volume doesn't seem to be serving any purpose.

Reply:
The reason for this mounted volume is because it seemed like an easy way to mimic as closely as possible the production environment, without requiring additional logic to create a directory in the container solely for test purposes. If you remove this line from the POM, the e2e test I've written will fail, because otherwise that `/csv` directory does not exist.

Reply:
I think it'd be good to have the app create the directory if it doesn't exist. That way someone could run it outside of a container if they wanted. Though I don't expect us to want to run it that way, I like the app not making assumptions about the environment in which its run.

Reply:
I agree, it should try to create the dir if it doesn't exist